Use sbt 1.5.1

eed3si9n opened the pull request.

eed3si9n committed:
Use sbt 1.5.1

typesafe-tools commented:
To validate this pull request against other sbt modules, please visit <A HREF="https://jenkins.scala-sbt.org:8592/job/sbt-validator/parambuild/?sbt=develop&zinc=pull/977/head&io=develop&librarymanagement=develop&util=develop&website=develop&deploy_to_bintray=false&run_sbt_community=false&verbose_dbuild=false&community_verbose=false">this link.</A>

eed3si9n committed:
Fork tests

```
[error] (zincJVM2_12 / Test / executeTests) java.lang.LinkageError: loader constraint violation in interface itable initialization: when resolving method "sbt.inc.ConstantBridgeProvider.fetchScalaInstance(Ljava/lang/String;Lxsbti/Logger;)Lxsbti/compile/ScalaInstance;" the class loader (instance of sbt/internal/FlatLoader) of the current class, sbt/inc/ConstantBridgeProvider, and the class loader (instance of sbt/internal/SbtInterfaceLoader) for interface xsbti/compile/CompilerBridgeProvider have different Class objects for the type xsbti/Logger used in the signature
```

eed3si9n committed:
Run Windows tests on GitHub Actions

eed3si9n committed:
Retry

Retry to workaround "The process cannot access the file because it is being used by another process." error.
It seems like RootPath#close does the actual writing of the zip file,
which sometimes fails.
